ui/admin/app/components/target-worker-filters/index.hbs

```diff
@@ -0,0 +1,45 @@
+<div class='target-worker-filters'>
+  <Hds::Form::Toggle::Field
+    name='target-worker-filter-toggle'
+    disabled={{@disabled}}
+    checked={{@checked}}
+    {{on 'change' @toggleAction}}
+    as |F|
+  >
+    <F.Label>{{t (concat 'resources.target.workers.' @name '.title')}}</F.Label>
+    <F.HelperText>{{t
+        (concat 'resources.target.workers.' @name '.description')
+      }}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  {{#if @toggleEnabled}}
+    <Hds::Form::TextInput::Field
+      disabled={{@disabled}}
```

Comment:
Would this ever be disabled?

Reply:
this is not needed, will remove

Reply:
This field would be disabled when viewing a target. On edit mode, it becomes editable